data/registry/instrumentation-go-ibm-sarama.yml

```
@@ -0,0 +1,16 @@
+# cSpell:ignore sarama
+title: IBM/sarama Instrumentation
+registryType: instrumentation
+language: go
+tags:
+  - sarama
+  - instrumentation
+  - go
+license: Apache 2.0
+description: Package sarama provides functions to trace the IBM/sarama package.
+authors:
+  - name: OpenTelemetry Authors
+  - name: Dominic Evans
+urls:
+  repo: https://github.com/dnwe/otelsarama
+createdAt: 2024-11-25
```

```suggestion
createdAt: 2024-11-25
package:
  name: github.com/dnwe/otelsarama
  registry: go
```

You may consider adding a `package` as well, this will provide a quick install note to your entry, see this example for reference: https://opentelemetry.io/ecosystem/registry/?s=xsam